monkey/infection_monkey/monkey.py

```diff
@@ -145,7 +146,7 @@ def start(self):
 
             self._exploiters = WormConfiguration.exploiter_classes
 
-            self._fingerprint = [fingerprint() for fingerprint in WormConfiguration.finger_classes]
+            self._fingerprint = get_fingerprint_instances()
```

Comment:
Want to leverage this PR to refactor `main` a bit?

Reply:
No